tools/travis/stage.sh

```diff
@@ -25,6 +25,7 @@ STAGE_BLINK_PLANNER="blink_planner"
 STAGE_CONNECTORS="connectors"
 STAGE_KAFKA_GELLY="kafka/gelly"
 STAGE_TESTS="tests"
+STAGE_CONNECTOR_HIVE_1="connector_hive_1"
```

Comment:
can we rename to "..._HIVE/hive_1.2.1" or "..._HIVE/hive_1_2_1"  to make it clean which hive 1.x version we are using?